x-pack/test/functional/services/ml/swim_lane.ts

```diff
@@ -208,5 +208,16 @@ export function SwimLaneProvider({ getService }: FtrProviderContext) {
       await testSubjects.click(`mlSwimLanePageSizePanel > ${rowsCount} rows`);
       await this.assertPageSize(testSubj, rowsCount);
     },
+
+    async waitForSwimLanesToload() {
```

Comment:
Nit - this should be `waitForSwimLanesToLoad` !

Reply:
Fixed in 435d210